File: QIC.EGame.GameMachineService/QIC.EGame.GameMachineService.GoalFootball/Dtos/ClientInitDto.cs
using System.Collections.Generic;
using QIC.EGame.GameMachineService.GoalFootball.Enums;

namespace QIC.EGame.GameMachineService.GoalFootball.Dtos
{
    /// <summary>
    /// 客户端初始化的Dto
    /// </summary>
    public class ClientInitDto
    {
        /// <summary>
        /// 投注基数
        /// </summary>
        public List<int> BaseAmounts { get; set; }

        /// <summary>
        /// 赔率信息
        /// </summary>
        public List<decimal> OddsInfo { get; set; }

        /// <summary>
        /// 用户余额
        /// </summary>
        public decimal Balance { get; set; }
    }
}

File: Client/Game_RoyalTreasure/src/Dto/GameCommand/ClientInitDto.ts
namespace Dto{
    /**
     * 初始化数据类
     */
    export class ClientInitDto{
        /**
         * 赔率信息
         */
        public OddsInfo:Object;
        /**
         * 余额
         */
        public Balance:number;
        /**
         * 最小投注额
         */
        public MinBet:number;
        /**
         * 最大投注额
         */
        public MaxBet:number;
    }
}